src/boot/efi/meson.build

```diff
@@ -162,7 +162,7 @@ foreach ctype : ['color-normal', 'color-entry', 'color-highlight', 'color-edit']
                 'EFI_' + c[1].strip().underscorify().to_upper()))
 endforeach
 
-if get_option('sbat-distro') != ''
+if get_option('sbat-distro') != '' and (not meson.is_cross_build() or get_option('sbat-distro') != 'auto')
```

Comment:
Hm? I do not understand why 'auto' is excluded here...

Reply:
We already enforce that all relevant sbat entries must be supplied to meson instead of auto detected inside this branch. But we want to skip this if we are cross building when sbat-distro was set to the new default (`auto`).